docs/build/js-client/3-setting-up.md

```diff
@@ -1,94 +1,27 @@
 # Setting up project
 
-## Install dependencies
+## CLI install
 
-Initialize an empty `npm` package:
+The best way to initialize the project is by using CLI
 
-```sh
-npm init
-```
-
-In newly created `package.json` file add module entry
-```json
-{
-  "type": "module",
-  ...
-}
-```
-
-We will need JS-client itself
-
-```sh
-npm install @fluencelabs/js-client@unstable
-```
+> You can find an installation guide [here](https://github.com/fluencelabs/cli).
 
-We will need the Fluence CLI to use the Aqua compiler, but only as a development dependency:
+Run the command below and follow instruction.
 
 ```sh
-npm install --save-dev @fluencelabs/cli
+fluence init -t ts --env kras 
```

Comment:
do think this is helpful? i'd rather see the user follow the options or see the dir name included, e.g., `fluence init -t ts --env kras jsc-example`